site/_plugins/download_filter.rb

```diff
@@ -43,27 +43,33 @@ def url_at_releases(platform, filename, ext)
       "#{RELEASES_URL_PREFIX}/manageiq-#{platform}-#{filename}.#{ext}"
     end
 
-    def file_size_from_url(url)
-      return "NA" if ENV["SKIP_FILE_SIZES"]
-
+    def file_info_from_url(url)
+      return no_file_info if ENV["SKIP_FILE_INFO"]
       uri = URI(url)
       Net::HTTP.start(uri.host, uri.port) do |http|
         response = http.request_head(uri.path)
         if response.code != '200'
-          Jekyll.logger.error("Jekyll::DownloadFilter.file_size_from_url(#{url})", "code=#{response.code}, message=#{response.message}")
-          return "NA"
+          Jekyll.logger.error("Jekyll::DownloadFilter.file_info_from_url(#{url})", "code=#{response.code}, message=#{response.message}")
+          return no_file_info
         end
+
         file_size = response['content-length'].to_f
-        return ActiveSupport::NumberHelper.number_to_human_size(file_size)
+        info = { "size" => ActiveSupport::NumberHelper.number_to_human_size(file_size) }
+        info["md5"] = response['etag'].delete('"')
```

Comment:
Style suggestion: make the 2 setters symmetrical.  Something like:
```
info = {
  "size" => ActiveSupport::NumberHelper.number_to_human_size(file_size),
  "md5" => response['etag'].delete('"')
}
```

or 

```
info = {}
info["size"] = ActiveSupport::NumberHelper.number_to_human_size(file_size)
info["md5"] = response['etag'].delete('"')
```

Reply:
Updated.